tests/test_utils/test_timer.py

```diff
@@ -0,0 +1,46 @@
+# tests.test_utils.test_timer
+# Tests for the stand alone timer functions in Yellowbrick utils.
+#
+# Author:   ZJ Poh <poh.zijie@gmail.com>
+# Created:  Tue Jul 17 21:11:11 2018 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+"""
+Tests for the stand alone timer functions in Yellowbrick utils.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from yellowbrick.utils.timer import *
+
+##########################################################################
+## Helper Function Tests
+##########################################################################
+
+class TestTimer(unittest.TestCase):
+    """
+    Timer functions and utilities
+    """
+
+    @mock.patch('time.time', mock.MagicMock(return_value=1234.))
+    def test_timer(self):
+        with Timer() as timer:
+            timer.time = mock.MagicMock(return_value=1236.)
+        self.assertIsInstance(timer.interval, float)
+        self.assertEqual(timer.interval, 2)
+
+
+    def test_human_readable_time(self):
+        self.assertEqual(human_readable_time(1.01), '00:00:01.0100')
+        self.assertEqual(human_readable_time(61.01), '00:01:01.0100')
+        self.assertEqual(human_readable_time(3661.01), '01:01:01.0100')
+        self.assertEqual(human_readable_time(360061.01), '100:01:01.0100')
```

Comment:
This is not a problem at all, and is kind of a stylistic thing; you don't have to change it if you don't want to. One thing we've done in yellowbrick tests, is to use table driven tests, to make it easy to add new test cases if something comes up; e.g. 

```python
tests = [
    (1.01, '00:00:01.0100'), 
    (61.01, '00:01:01.0100'), 
    ... 
]

for case, expected in tests:
    assert human_readable_time(case) == expected 
```

Reply:
Note that in these tests, you can also use something like `pytest.parameterize` -- but don't worry about that!

Reply:
I will look into pytest.parametrize~